src/Restaurant.js

```diff
@@ -1,14 +1,13 @@
 import React from "react";
+import Order from "./Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
     <div>
```

Comment:
As a general rule use only as much HTML as you need, so unless it's for styling purposes, no need to wrap things in divs.